### PR TITLE
bug-fix: Do not support MFM in super or broadcast channels.

### PR DESCRIPTION
**Pull request by liamcho**
Do not support MFM in super or broadcast channels.

Fixes: [CLNP-1045](https://sendbird.atlassian.net/browse/CLNP-1045)

[CLNP-1045]: https://sendbird.atlassian.net/browse/CLNP-1045?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ui/MessageInput/utils.js [hunk 1]
```diff
@@ -38,4 +38,10 @@ export const nodeListToArray = (childNodes) => {
   }
 };
 
-export default debounce;
+export function isChannelTypeSupportsMultipleFilesMessage(channel) {
+  return channel
+    && typeof channel.isGroupChannel === 'function'
+    && channel.isGroupChannel()
```
Comment (AhyoungRyu) [inline src/ui/MessageInput/utils.js:44]: nit; 
I think those two lines can be simplified like below. 
```suggestion
    && channel?.isGroupChannel?.()
```

File: src/ui/MessageInput/utils.js [hunk 2]
```diff
@@ -38,4 +38,10 @@ export const nodeListToArray = (childNodes) => {
   }
 };
 
-export default debounce;
```
Comment (AhyoungRyu) [inline src/ui/MessageInput/utils.js:41]: Doesn't it have any references?

File: src/ui/MessageInput/utils.js (hunk 1 above)
Comment (liamcho) [reply src/ui/MessageInput/utils.js:44]: oh thanks! `&& channel?.isGroupChannel?()` I tried this and didnt work but now I understand `.` was missing.

File: src/ui/MessageInput/utils.js (hunk 2 above)
Comment (liamcho) [reply src/ui/MessageInput/utils.js:41]: no reference so I removed it

**Review by AhyoungRyu — APPROVED** [review]
👍

**Pull request merged**